GitVersion.yml

```diff
@@ -20,5 +20,5 @@ branches:
     - feature
     - support
     - hotfix
-next-version: "3.1"
```

Comment:
Is this still used? I thought our builds worked off tags these days?

Reply:
If you only want the patch number to increment, it all happens automatically. But this is the only way I know to signify that you're working on the next minor or major version.

The main practical reason I do this is that this setting determines the version number that is produced while you're still on a feature branch. I set this so that when building preview packages for test purposes, they come out as 3.2.x-previewTag.N. If you don't do this, they'd be 3.1.x-previewTag.N, making NuGet think that these preview builds are actually older than the latest non-preview.

Also, this means you get a record in the commit history of when the version number changed. (The problem with using tags alone for that is that tags can be moved.)